setup_codecov.sh

```diff
@@ -1,14 +1,37 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
 
 CODECOV_VERSION=0.6.1
 
+# Determine the Architecture to install the Codecov uploader.
+if [[ -z "${DD_TARGET_ARCH}" ]]; then
+  echo "DD_TARGET_ARCH environment variable is not set. The Codecov x64 uploader will be installed by default."
+  CODECOV_ARCH="linux"
+else
+  case $DD_TARGET_ARCH in
+    aarch64)
+      CODECOV_ARCH="aarch64"
+      ;;
+    armhf)
+      CODECOV_ARCH="aarch64"
+      ;;
```

Comment:
```suggestion
    aarch64|armhf)
      CODECOV_ARCH="aarch64"
      ;;
```

IIRC, but I'm a bit rusted so try it before :)

Reply:
Nice ! Didn't know this syntax ! Thank you